command/volume_status_host_test.go

```diff
@@ -161,7 +161,7 @@ capability {
 	code = cmd.Run(args)
 	must.Eq(t, 1, code)
 	must.StrContains(t, ui.ErrorWriter.String(),
-		"Error listing volumes: no volumes with prefix or ID")
+		"Error listing host volumes: no volumes with prefix or ID")
```

Comment:
I know this is a tangentially-related test, but super nit: `N`o volumes vs `n`o volumes -- from your example output:

```
$ nomad volume status not-existing
Error listing host volumes: no volumes with prefix or ID "not-existing" found
Error listing CSI volumes: No volumes with prefix or ID "not-existing" found
```

I would never have noticed, except for them being right next to each other now. :yum:

Reply:
Will fix. The inconsistency is a result of the inner error for CSI coming from a shared function far away from here, which we couldn't use for host volumes. But ideally we always have lower-case errors in helper functions so the final output in `c.Ui.Error` can have the capital letter.

Reply:
Oh man, I broke like a zillion CLI tests that expect that to be capitalized. :facepalm: Lemme go hunt all those down.

Reply:
:lolsob: sorry!

Reply:
Oh :relieved: it only looked like a zillion because of the retries on GHA. Just two! Fixed!